core/src/components/datetime/datetime.tsx

```diff
@@ -308,58 +308,77 @@ export class Datetime implements ComponentInterface {
    */
   @Prop() firstDayOfWeek = 0;
 
+  /**
+   * If `true`, multiple dates can be selected at once. Only
+   * applies to `presentation="date"` and `preferWheel="false"`.
+   */
+  @Prop() multiple = false;
```

Comment:
~ Noticed that 1/2 of our implementations have a default value and the other set the value to `false`. Unsure if any action is needed here, but we should probably land on consistency in the next major.

For example:
- AccordionGroup, Input - multiple is unset
- Select - multiple defaults to false 
 
This does affect the type signatures that are generated, but I am unsure of the impact of the field being marked as optional vs. required from the devs perspective.